mkosi/distributions/gentoo.py

```diff
@@ -329,50 +307,18 @@ def merge_user_pkgs(self) -> None:
         if self.state.config.packages:
             self.invoke_emerge(pkgs=self.state.config.packages)
 
-
     def invoke_emerge(
         self,
-        inside_stage3: bool = True,
         pkgs: Sequence[str] = (),
         actions: Sequence[str] = (),
         opts: Sequence[str] = (),
     ) -> None:
-        if not inside_stage3:
-            from _emerge.main import emerge_main  # type: ignore
-
-            PREFIX_OPTS: list[str] = []
-            if "--sync" not in actions:
-                PREFIX_OPTS = [
-                    f"--config-root={self.root.resolve()}",
-                    f"--root={self.root.resolve()}",
-                    f"--sysroot={self.root.resolve()}",
-                ]
-
-            MkosiPrinter.print_step(f"Invoking emerge(1) pkgs={pkgs} "
-                                    f"actions={actions} outside stage3")
-            emerge_main([*pkgs, *opts, *actions] + PREFIX_OPTS + self.emerge_default_opts)
-        else:
-            cmd = ["/usr/bin/emerge", *pkgs, *self.emerge_default_opts, *opts, *actions]
-
-            MkosiPrinter.print_step("Invoking emerge(1) inside stage3"
-                                    f"{self.root}")
+        cmd = ["/usr/bin/emerge", *pkgs, *self.emerge_default_opts, *opts, *actions]
+        run_workspace_command(self.state, cmd, network=True)
 
-            bwrap = [
-                "--bind", self.portage_cfg['PORTDIR'], self.portage_cfg['PORTDIR'],
-                "--bind", self.portage_cfg['DISTDIR'], self.portage_cfg['DISTDIR'],
-                "--bind", self.portage_cfg['PKGDIR'],  self.portage_cfg['PKGDIR'],
-            ]
-            run_workspace_command(self.state, cmd, network=True, bwrap_params=bwrap)
-
-    def _dbg(self, state: MkosiState) -> None:
+    def _dbg(self) -> None:
         """this is for dropping into shell to see what's wrong"""
-
-        bwrap = [
-            "--bind", self.portage_cfg['PORTDIR'], self.portage_cfg['PORTDIR'],
-            "--bind", self.portage_cfg['DISTDIR'], self.portage_cfg['DISTDIR'],
-            "--bind", self.portage_cfg['PKGDIR'],  self.portage_cfg['PKGDIR'],
-        ]
-        run_workspace_command(self.state, ["sh"], network=True, bwrap_params=bwrap)
+        run_workspace_command(self.state, ["sh"], network=True)
```

Comment:
Can we drop this? We already have --debug run and if needed you can always just add this locally since its a one liner

Reply:
done